parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -103,7 +103,8 @@ impl Command {
 								Parent,
 								GlobalConsensus(network),
 								origin_location,
-							).into(),
+							)
+								.into(),
```

Comment:
Is the formatting right? It looks a little weird. 🤔

Reply:
Yes, it's formatted by the [rule](https://github.com/Snowfork/snowbridge/blob/main/parachain/rustfmt.toml). Notice sometimes we've format inconsistent which makes cooperating development/code review a bit challenge. 

So would suggest as a team to always share the same formatting rule. Btw: config the ide with [auto format ](https://github.com/rust-lang/rust-analyzer/issues/3916#issuecomment-611617573) could be helpful.